scripts/umbrel-os/status-server/static/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Umbrel</title>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Are we still compatible with this legacy browser?

Reply:
It's just pulled straight from https://github.com/getumbrel/umbrel-dashboard/blob/c36af7cb58bec8d0533178675886a63f4a1fcc88/public/index.html